scripts/publish-native-packages.sh

```diff
@@ -53,5 +53,5 @@ done
 
 for f in "$source_packages_path"/*.src.rpm; do
   rpmsign --define="%_gpg_name Serokell <tezos-packaging@serokell.io>" --define="%__gpg $(which gpg)" --addsign "$f"
-  /run/wrappers/bin/sudo -u copr-uploader /run/current-system/sw/bin/copr-cli build "$copr_project" --nowait "$f"
+  /run/wrappers/bin/sudo -u copr-uploader /run/current-system/sw/bin/copr-cli build "$copr_project" "${CHROOTS:-}" --nowait "$f"
```

Comment:
Last time I tried `--chroots` it didn't work for me on the version from our nixpkgs fork:(